timesketch/ui/graphs/graph-view.component.ts

```diff
@@ -31,12 +31,56 @@ export class GraphViewComponent {
 
   constructor(private readonly changeDetectorRef: ChangeDetectorRef) {}
 
+  showNeighborhood(event) {
+    let neighborhoodCollection = event.cy.collection()
+    const selectedElements = event.cy.filter(':selected')
```

Comment:
The same applies, could be just `selected`.

Reply:
Done.